src/main/java/com/labsynch/labseer/chemclasses/bbchem/ChemStructureServiceBBChemImpl.java

```diff
@@ -1234,6 +1234,7 @@ public StandardizationSettingsConfigCheckResponseDTO checkStandardizerSettings(S
 
 
 	@Transactional
+	@Override
```

Comment:
I noticed this didn't have @ Override.  It doesn't have any practical affect but its good practice to annotate the overrides methods so I added it to this PR.